step-templates/clone-tenant.json

```diff
@@ -50,6 +50,14 @@
       "DisplaySettings": {
         "Octopus.ControlType": "Checkbox"
       }
+    },
```

Comment:
@jburger wouldn't the cloning be for the space you are in ? I don't think we should be cloning tenants across space boundaries. Hence I am not sure we need this field ?

Reply:
Thanks @johnsimons - good pickup! I believe this was discussed and you are right, we don't want to do this.